packages/manager/modules/pci/src/projects/project/storages/containers/containers.controller.js

```diff
@@ -26,7 +26,7 @@ export default class PciStoragesContainersController {
               : 'pci_projects_project_storages_containers_toggle_public_succeed',
             { name: container.name },
           ),
-          'pci.projects.project.storages.containers.container',
+          'pci.projects.project.storages.containers.containers',
```

Comment:
`containers.containers` so we have a list of list ? Hard to know what to expect under this state

Reply:
It is just `pci.projects.project.storages.containers`. ill fix this